samples/drivers/led_ws2812/README.rst

```diff
@@ -102,6 +102,8 @@ This sample uses different drivers depending on the selected board:
 I2S driver:
 
 - thingy52_nrf52832
+- nrf5340dk_nrf5340 (1.8V logic level, make sure to use a logic level shifter!)
+    - should work for other boards featuring an nRF5340 host processor
```

Comment:
This is incorrect, the nrf5340dk is 3.3v logic level

Reply:
Fixed it